pkg/config/config.go

```diff
@@ -297,8 +299,8 @@ func New() *Config {
 	}
 
 	// TODO (timothysc) reference the other consts
-	cfg.WorkerImage = "gcr.io/heptio-images/sonobuoy:latest"
-	cfg.ImagePullPolicy = "Always"
+	cfg.WorkerImage = DefaultImage
+	cfg.ImagePullPolicy = DefaultSonobuoyPullPolicy
```

Comment:
Since the const `DefaultImage` uses `gcr.io/heptio-images/sonobuoy:buildinfo.Version`, I think the pull policy should change to `IfNotPresent` so that if it's already there, it won't always attempt to pull (especially in air-gapped envs).

Reply:
Done.